src/Materials/PBR/pbrBaseMaterial.ts

```diff
@@ -1745,6 +1762,7 @@ export abstract class PBRBaseMaterial extends PushMaterial {
         // Matrices
         if (!defines.INSTANCES || defines.THIN_INSTANCES) {
             this.bindOnlyWorldMatrix(world);
+            this.prePassConfiguration.bindForSubMesh(this._activeEffect, scene, mesh, world, this.isFrozen);

```

Comment:
Are you sure this should not be done for instances?

We don't pass the world matrix for instances because it's `world0/world1/world2/world3` which is used instead, but I do think the prepass config should be done for instances, as the velocity shader code is also applied in the instance case.

Reply:
you are definitely right on that, but as of today, velocity (and therefore motion blur) is broken with instances, since we only calculate velocity from the main world matrix.  
This motion blur + instances issue should be fixed in another PR.

Anyway, in the meantime, I have put this line out of the if statement